icefoganalytics/sfa-client
# Enhancement: Add Parents Sections to Student Application Endpoint Response

Relates to https://github.com/icefoganalytics/student-financial-aid/issues/2
Fixes https://github.com/icefoganalytics/sfa-client/issues/36

# Context

As part of the "view submitted applications and their current status" project, this task covers returning the parent data.

# Implementation

I'm preferring parent data retrieved through the `student_persons` table, and if that doesn't exist, I'm loading data from the `application.parent1_id` and `application.parent2_id` fields.

# Data Format

```json
{
  "id": "some-application-id",
  "parents": {},
}
```

# Testing Instructions

1. Boot the `sfa-client` repo via `API_PORT=3100 dev up`
2. Boot the `student-financial-aid` front- and back-ends.
3. Log in to the app at http://localhost:8080/
4. Go to http://localhost:3000/api/portal/students/8634/applications/1995 and you will see the empty parents section.
5. Go to http://localhost:3000/api/portal/students/1470/applications/256 and you will see some parents values.
